core/arch/arm/plat-stm32mp1/scripts/stm32image.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
```

Comment:
This script has a number of small formatting issues, could you run `pep8` and see if you can address most of the warnings/errors?

Reply:
ok. thanks

Reply:
Please also make sure it works with Python3 (Python 2 have [no maintenance in 2020](https://pythonclock.org/)). So, maybe even better to have `#!/usr/bin/env python3`

Reply:
ok